README.md

```diff
@@ -84,6 +85,25 @@ manifests:
       # which to fetch artifacts. Defaults to the last step.
 ```
 
+### Using GCS as an artifact source
+
+You can use dbt-loom to fetch manifest files from Google Cloud Storage by setting up a `gcs` manifest in your `dbt-loom` config.
+
+```yaml
+manifests:
+  - name: project_name
+    type: gcs
+    config:
```

Comment:
Since `project_id` is specified in the config object, here's an addition to the README to ensure that it's part of the docs.

```suggestion
    config:
      project_id: <YOUR GCP Project ID>
      # The alphanumeric ID of the GCP project that contains your target bucket.

```